### ISSUE TITLE
Electron app tray icon has disappeared

### ISSUE THREAD

**Issue by vyaas** [kickscondor/fraidycat#125]
**Before you start.**

* [x ] I have checked the [Common Problems](https://github.com/kickscondor/fraidycat/wiki/Common-Problems) page.
* [ x] I have searched the issue tracker for a bug that matches the one I want to file, without success.

**Desktop (please complete the following information):**
 - OS: Linux
 - Browser: N/A; using electron app
 - Version 1.1.3
 - URL you are attempting to follow [if applicable]

**Describe the bug**

Fraidycat's electron app's tray icon has disappeared after a recent update. Upon closing fraidycat, it still shows as running on ``top``.

**Comment by kickscondor** [comment]
I'm getting this as well. This appears to be due to the upgrade to Electron 8.

This purports to be a solution: https://github.com/zulip/zulip-desktop/issues/910#issuecomment-608056150

**Comment by Petemir** [comment]
Hi,

It worked for me in 1.1.3, with Ubuntu 18.04 KDE... maybe it's possible to add it as an option, disabled by default?

**Comment by kickscondor** [comment]
Yes I hope to do something like that soon - thank you for speaking up! I'll reopen for now.

**Comment by jaybosamiya** [comment]
On Debian Stable (i.e. Debian Buster), the tray icon broke when moving from 1.1.2 to 1.1.3 (I tried upgrading to 1.1.4, saw that it broke, so I downgraded to 1.1.3 seeing this issue, and was still broken there). I am not sure if something related to this changed in 1.1.3, but if/when you do add it as an option, could you please have the option work like 1.1.2 for the system tray? Thanks so much! :)
